src/WarcraftLegacies.Source/Setup/GameSetup.cs

```diff
@@ -41,7 +41,6 @@ public static void Setup()
       new PlayerSetup(preplacedUnitSystem, allLegendSetup, artifactSetup).Setup();
       FactionChoiceDialogSetup.Setup(preplacedUnitSystem, artifactSetup, allLegendSetup);
       SharedQuestSetup.Setup(preplacedUnitSystem, artifactSetup, allLegendSetup);
-      ObserverSetup.Setup(new[] { Player(21) });
```

Comment:
Wondering... Does this mean we don't have any observer slots now? Is that a problem?

Reply:
Yes

Reply:
Ok - that's an unnecessary regression right? So I think we either need to move the Observer slot somewhere else as part of this PR, or create a ticket for it under the "x. Floating" header in the Old Gods project.